lib/utils/index.js

```diff
@@ -29,9 +30,17 @@ const hasChanged = ({ accessor }, newValue, oldValue) => {
   return newValue !== oldValue;
 };
 
+const cleanModel = model => mapValues(model, field => {
+  if (typeof field === 'string') {
+    return striptags(field);
+  }
+  return field;
```

Comment:
We possibly want to traverse down through arrays and nested objects to clean _all_ strings rather than just top-level values.